scripts/org.ooni.probe.cli.plist

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>Label</key>
+    <string>org.ooni.probe.daily-run</string>
+
+    <key>KeepAlive</key>
+    <false/>
+    <key>RunAtLoad</key>
+    <true/>
+
+    <key>Program</key>
+    <string>/Users/USERNAME/.ooniprobe/ooniprobe</string>
+    <key>ProgramArguments</key>
+    <key>ProgramArguments</key>
+    <array>
+        <string>--config="/Users/USERNAME/.ooniprobe/config-100sites.json"</string>
```

Comment:
Does this exists by default?

Reply:
Nope, as mentioned in the commit you should make a special config for this automatic test which limits to 100 websites, so that it's more likely to converge in 1h.